src/main/java/tc/oc/pgm/Config.java

```diff
@@ -408,4 +408,18 @@ public static boolean overwriteExisting() {
       return getConfiguration().getBoolean("sidebar.overwrite", false);
     }
   }
+
+  public static class Discord {
+    public static boolean enabled() {
+      return getConfiguration().getBoolean("discord.enabled");
```

Comment:
Add `false` default to this method using the second optional param (see line 408 for an example)